src/Nethermind/Nethermind.Init.Snapshot/ISnapshotConfig.cs

```diff
@@ -15,4 +15,10 @@ public interface ISnapshotConfig : IConfig
 
     [ConfigItem(Description = "The SHA-256 checksum of the snapshot file.", DefaultValue = "null")]
     public string? Checksum { get; set; }
+
+    [ConfigItem(Description = "Directory where snapshot will be stored.", DefaultValue = "snapshot")]
+    public string SnapshotDirectory { get; set; }
+
+    [ConfigItem(Description = "Snapshot file name.", DefaultValue = "snapshot.zip")]
+    public string SnapshotFileName { get; set; }
```

Comment:
let's take the SnapshotFileName from the last path segment in the snapshot url, wdyt?

Reply:
It should resume downloading without problems. If you use different config params

Reply:
yeah, but you need to change the `SnapshotFileName` if you change the `DownloadUrl`, not a big deal I guess